### PR TITLE
Idex 3006

### PR DESCRIPTION
**Pull request by vparfonov**

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/ProjectListeners.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
     private final File workspace;
 
     @Inject
-    public ProjectListeners(@Named("che.workspace.path") String workspacePath, EventService eventService) {
+    public ProjectListeners(@Named("vfs.local.fs_root_dir") String workspacePath, EventService eventService) {
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/ProjectListeners.java]: che.projects.root

File: plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/java/org/eclipse/che/ide/ext/java/BaseTest.java [hunk 2]
```diff
@@ -34,9 +34,12 @@
     protected static Map<String, String> options = new HashMap<>();
     protected static JavaProject project;
     protected static final String wsPath = BaseTest.class.getResource("/projects").getFile();
-    protected static ResourcesPlugin plugin = new ResourcesPlugin(wsPath + "/index", BaseTest.class.getResource("/projects").getFile());
-    protected static JavaPlugin javaPlugin = new JavaPlugin(wsPath + "/set");
-    protected static FileBuffersPlugin fileBuffersPlugin = new FileBuffersPlugin();
+    private static final   String workspacePath = BaseTest.class.getResource("/projects").getFile();
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/java/org/eclipse/che/ide/ext/java/BaseTest.java]: che.projects.root

File: plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/java/org/eclipse/che/ide/ext/java/ProjectManagerMock.java [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author Evgen Vidolob
+ */
+public class ProjectManagerMock implements ProjectManager {
+
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/java/org/eclipse/che/ide/ext/java/ProjectManagerMock.java]: This class is not a mock. May be dummyProjectManager but for sure not a mock

Comment (vparfonov) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/java/org/eclipse/che/ide/ext/java/ProjectManagerMock.java]: ok

File: plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/ProjectListeners.java (hunk 1 above)
Comment (vparfonov) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/ProjectListeners.java]: will be fixed in https://jira.codenvycorp.com/browse/IDEX-3190

**Comment by skabashnyuk** [comment]
LGTM

**Pull request merged**